dht_test.go

```diff
@@ -922,12 +926,12 @@ func TestUnfindablePeer(t *testing.T) {
 	connect(t, ctx, dhts[2], dhts[3])
 
 	// Give DHT 1 a bad addr for DHT 2.
-	dhts[1].host.Peerstore().ClearAddrs(peers[2])
-	dhts[1].host.Peerstore().AddAddr(peers[2], maddrs[0], time.Minute)
+	dhts[1].host.Peerstore().ClearAddrs(dhts[2].PeerID())
+	dhts[1].host.Peerstore().AddAddr(dhts[2].PeerID(), dhts[0].Host().Addrs()[0], time.Minute)
```

Comment:
Why are `.host` and `.Host()` used inconsistently? Not just here but throughout the tests.

Reply:
Just that I've exposed `Host` in this PR, but existing code is using `host`. Both are fine, depending on the situation, and `host` is accessible here as the tests have visibility into the internals of this package.